utils/wp-cli-1.1.0.spec

```diff
@@ -0,0 +1,45 @@
+%define Source1 /home/`whoami`/rpmbuild/SOURCES/wp.1.gz
```

Comment:
Could we rename this file to `utils/wp-cli-rpm.spec`, so we don't need to rename it in each release?